src/ml_gym/frontend/dashboard/src/features/globalConfig/globalConfig.tsx

```diff
@@ -0,0 +1,39 @@
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { FilterConfigType } from "../../app/datatypes"
+
+
+type GlobalConfigPropsType = {
+    filterConfig: FilterConfigType
+    setFilterConfig: (filterConfig: FilterConfigType) => void
+}
+
+const GlobalConfig: React.FC<GlobalConfigPropsType> = ({ filterConfig, setFilterConfig }) => {
+
+
+    const handleChange = (event: { target: { value: any; }; }) => {
+        setFilterConfig({...filterConfig, tmpMetricFilterRegex: event.target.value});
```

Comment:
Codacy has a fix for the issue: [Replace `········setFilterConfig({...filterConfig,·tmpMetricFilterRegex:·event.target.value});` with `····setFilterConfig({⏎······...filterConfig,⏎······tmpMetricFilterRegex:·event.target.value,`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
    setFilterConfig({
```